server/lib/template/helpers/objects.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/**
+ * @function look
+ * @description look value into an object given
+ * @param {object} obj The concerned object
+ * @param {string} property The object property
+ * @param {string} property2 The object property
+ */
+function look(obj, property, property2) {
+  // Missing parameter take the function name as name property
+  // if property2 is missing, it will be an object with an attribute name: 'function name'
+  return obj && property && property2 && property2.name ? obj[property] :
```

Comment:
In both checks, you do this:
```js
obj && property && property2
```
I would cache this boolean for code clarity.  Like this:
```js
let hasProperInput = obj && property && property2;
let hasSecondProperty = hasProperInput && !property2.name;
return hasProperInput ?  obj[property] :
    hasSecondProperty ? obj[property][property2] :
    '';
```

This won't block the PR.